src/main/scala-templates/default/com/normation/plugins/datasources/EnablePluginImpl.scala

```diff
@@ -37,17 +37,14 @@
 
 package com.normation.plugins.datasources
 
-import bootstrap.rudder.plugin.CheckRudderPluginDatasourcesEnable
-import bootstrap.rudder.plugin.DatasourcesStatus
+import com.normation.plugins.PluginStatus
+import com.normation.plugins.PluginStatusInfo
 
 /*
- * The class will be loaded by ServiceLoader, it needs an empty constructor.
+ * unlimited version of the plugin 
  */
-
-final class CheckRudderPluginDatasourcesEnableImpl() extends CheckRudderPluginDatasourcesEnable {
-    val hasLicense = false
-    val isEnabled = true
-    val enabledStatus = DatasourcesStatus.Enabled
-    val licenseInformation = None
+final class CheckRudderPluginDatasourcesEnableImpl() extends PluginStatus {
```

Comment:
Why not using: AlwaysEnabledPluginStatus instead ?

Reply:
and removing this file (and removing lines from pom.xml if default is now useless

Reply:
because we need the correct name to be able to instanciate that service in the plugin module. I don't have any simple way to make that work without a full-blown IoC or Java Service arch. I don't want to use them.

Reply:
All right !!